src/marketplace/sellers/product-descriptions.md

```diff
@@ -96,4 +96,4 @@ _Example 3:_
 
 ATTENTION: Use this extension at your own risk. Sensitive customer payment data is stored on your server. Merchants are expected to take every precaution that they are fully PCI certified. Please only purchase this extension if you have an environment where storing credit card information is permitted by law.
 
-[1]: https://marketplacesupport.magento.com/hc/en-us
\ No newline at end of file
+[1]: https://marketplacesupport.magento.com/hc/en-us
```

Comment:
Not sure why this line (`#99`) shows up as a change; it looks identical to me! 😸

Reply:
I don't know why this line highlighted ))